app/experimenter/experiments/tests/test_models.py

```diff
@@ -2095,6 +2095,18 @@ def test_latest_review_request_returns_change_for_idle_to_review(self):
 
         self.assertEqual(experiment.changes.latest_review_request(), change)
 
+    def test_latest_review_request_returns_change_for_dirty_to_review(self):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LIVE_DIRTY,
+        )
+
+        experiment.publish_status = NimbusExperiment.PublishStatus.REVIEW
+        experiment.save()
```

Comment:
There could be another lifecycle specifically for being in the live/dirty > review state?

Reply:
There is, I could use [LIVE_REVIEW_REQUESTED](https://github.com/mozilla/experimenter/blob/bb461d14e599870808c0e538e75c0c601c71ff05/app/experimenter/experiments/tests/factories.py#L223) instead 👍

Reply:
Actually, I think that I'll keep these as is. These tests are creating an experiment one step before we want it (in this case, it's a Live Dirty rollout) and then we're manually generating the change log so we can assert on it.

If we created the lifecycle directly, we'd have to do a bunch of fun matching with the way that the [factory generates changelogs](https://github.com/mozilla/experimenter/blob/bb461d14e599870808c0e538e75c0c601c71ff05/app/experimenter/experiments/tests/factories.py#L468). I think this is simpler and easier to understand